test/commands/add/nickname_spec.rb

```diff
@@ -7,6 +7,15 @@
 
   let(:content) { CONTENT }
 
+  describe "when friend name and nickname are blank" do
```

Comment:
Just to double-check: my read is that we need both of these to be blank because if only one is blank we'll just be repeating the `"when nickname is blank"` test below. Is that right?

Reply:
Yes, I implemented it with that intention.